### PR TITLE
Add ELK documentation

### PR DESCRIPTION
**Pull request by Vadorequest**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: 
- https://github.com/reaviz/reaflow/issues/4

## What is the new behavior?

Better documentation regarding ELK ("getting started") and default Reaflow configuration

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

⚠️ Depends on https://github.com/reaviz/reaflow/pull/63 _(it needs to be merged before merging this PR)_

### CODE REVIEW

**Comment by Vadorequest** [comment]
@amcdnl This PR is "ready" for a first review. I consider it to be a draft, still. I wouldn't be surprised if you'd change things around.

Needs #63 to be merged beforehand.
Shouldn't be merged quite yet.

File: docs/Advanced/ELK.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,224 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Docs/Advanced/Eclipse Layout Kernel" />
```
Comment (amcdnl) [inline docs/Advanced/ELK.stories.mdx:3]: Let's move this under `Getting Started` as `Layout` and then go into the whole ELK thing.

File: docs/Advanced/ELK.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+# Eclipse Layout Kernel (ELK)
+
+[`ELK`](https://www.eclipse.org/elk/) is what's use by Reaflow to position the nodes and edges automatically.
+
+Actually, Reaflow uses [`ELKjs`](https://github.com/kieler/elkjs), which is a port of ELK to JavaScript.
+
+## Changing ELK behavior
+
+Unfortunately, the ELK/ELKjs documentation isn't TypeScript-friendly, and you'll need to dig into the ELK documentation and issues if you're trying to change **how the graph's layout behaves**.
```
Comment (amcdnl) [inline docs/Advanced/ELK.stories.mdx:13]: Avoid negative statements.

File: docs/Advanced/ELK.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+- [ELK official website](https://www.eclipse.org/elk/)
+- [ELK Demonstrators](https://rtsys.informatik.uni-kiel.de/elklive/index.html)
+- [Tool to convert `elkt <=> json` both ways](https://rtsys.informatik.uni-kiel.de/elklive/conversion.html)
+- [Tool to convert `elkt` to a graph](https://rtsys.informatik.uni-kiel.de/elklive/elkgraph.html)
+- [Java ELK implementation of the `layered` algorithm](https://github.com/eclipse/elk/tree/master/plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/p2layers)
+- [Community examples soure code](https://github.com/eclipse/elk-models/tree/master/examples) _(which are displayed on [ELK examples](https://rtsys.informatik.uni-kiel.de/elklive/examples.html))_
+- [Klayjs example](http://kieler.github.io/klayjs-d3/examples/interactive) (ELK is the successor of KlayJS and [should support the same options](https://github.com/kieler/elkjs/issues/122#issuecomment-777781503))
+- [ELKjs issues opened by Austin](https://github.com/kieler/elkjs/issues?q=is%3Aissue+sort%3Aupdated-desc+author%3Aamcdnl)
+
+> From experience, changing ELK behavior is tough, because it is hard to comprehend what option(s) should be changed to achieve one's goal.
```
Comment (amcdnl) [inline docs/Advanced/ELK.stories.mdx:27]: Avoid negative statements.

Comment (amcdnl) [reply docs/Advanced/ELK.stories.mdx:27]: Avoid using "I"

File: docs/Advanced/ELK.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+- [ELKjs GitHub](https://github.com/kieler/elkjs)
+- [ELK official website](https://www.eclipse.org/elk/)
+- [ELK Demonstrators](https://rtsys.informatik.uni-kiel.de/elklive/index.html)
+- [Tool to convert `elkt <=> json` both ways](https://rtsys.informatik.uni-kiel.de/elklive/conversion.html)
+- [Tool to convert `elkt` to a graph](https://rtsys.informatik.uni-kiel.de/elklive/elkgraph.html)
+- [Java ELK implementation of the `layered` algorithm](https://github.com/eclipse/elk/tree/master/plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/p2layers)
+- [Community examples soure code](https://github.com/eclipse/elk-models/tree/master/examples) _(which are displayed on [ELK examples](https://rtsys.informatik.uni-kiel.de/elklive/examples.html))_
+- [Klayjs example](http://kieler.github.io/klayjs-d3/examples/interactive) (ELK is the successor of KlayJS and [should support the same options](https://github.com/kieler/elkjs/issues/122#issuecomment-777781503))
+- [ELKjs issues opened by Austin](https://github.com/kieler/elkjs/issues?q=is%3Aissue+sort%3Aupdated-desc+author%3Aamcdnl)
```
Comment (amcdnl) [inline docs/Advanced/ELK.stories.mdx:25]: Call this out as "opened by reaflow team"

File: docs/Advanced/ELK.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+
+## Dragging and positioning nodes manually
+
+It's not currently possible to manually position a node in the layout, because ELK automatically position the elements depending on the selected `algorithm`.
+
+This is a known limitation of ELK:
+- [[ELK] - Tracking issue - Manually positioning the nodes ("Standalone Edge Routing")](https://github.com/eclipse/elk/issues/315)
+- [[Reaflow] - Tracking issue - Custom Dragging](https://github.com/reaviz/reaflow/issues/4)
+
+> TLDR; They're willing to implement the feature (AKA "Standalone Edge Routing"), but they're waiting for an open source contribution.
```
Comment (amcdnl) [inline docs/Advanced/ELK.stories.mdx:37]: I would reword this

**Review by amcdnl — CHANGES_REQUESTED** [review]
